tests/index/test_index_search_meilisearch.py

```diff
@@ -458,12 +456,12 @@ def test_attributes_to_search_on_search_no_match(index_with_documents):
     )
     assert response["hits"] == []
 
+# Commented because of https://github.com/meilisearch/meilisearch-python/issues/901
+# @pytest.mark.usefixtures("enable_vector_search")
+# def test_vector_search(index_with_documents_and_vectors):
+#     response = index_with_documents_and_vectors().search(
+#         "How to Train Your Dragon", opt_params={"vector": [0.1, 0.2]}
+#     )
 
-@pytest.mark.usefixtures("enable_vector_search")
-def test_vector_search(index_with_documents_and_vectors):
-    response = index_with_documents_and_vectors().search(
-        "How to Train Your Dragon", opt_params={"vector": [0.1, 0.2]}
-    )
-
-    assert response["hits"][0]["id"] == "287947"
-    assert response["vector"] == [0.1, 0.2]
+#     assert response["hits"][0]["id"] == "287947"
+#     assert response["vector"] == [0.1, 0.2]
```

Comment:
I would recommend instead xfailing the test instead as a reminded that it needs to be fixed.

```suggestion
@pytest.mark.xfail(strict=True, reason="https://github.com/meilisearch/meilisearch-python/issues/901")
@pytest.mark.usefixtures("enable_vector_search")
def test_vector_search(index_with_documents_and_vectors):
    response = index_with_documents_and_vectors().search(
        "How to Train Your Dragon", opt_params={"vector": [0.1, 0.2]}
    )

    assert response["hits"][0]["id"] == "287947"
    assert response["vector"] == [0.1, 0.2]

```

Though commenting it certainly works also, so if you would rather stay with that no worries.

Reply:
good idea!

Reply:
I cannot accept your suggestion 😢

Reply:
Interesting. Want me to push it?

Reply:
Yes you can do it 😊

Reply:
Alright, done :smile: 

Note that the tests failed on the non-rc tests because I made it a strict xfail meaning if the test passes it will mark as failed. This way you know you can remove the xfail if the test starts passing.